src/languages/en-US/commands/social.json

```
@@ -1,9 +1,13 @@
 {
 	"autoRoleUpdateConfigured": "This role is already configured as an autorole. Use the remove type instead.",
 	"autoRoleUpdateUnconfigured": "This role is not configured as an autorole. Use the add type instead.",
-	"autoRoleUpdate": "Updated autorole: {{role.name}} ({{role.id}}). Points required: {{points}} (before: {{before}})",
-	"autoRoleRemove": "Removed the autorole: {{role.name}} ({{role.id}}), which required {{before}} points.",
-	"autoRoleAdd": "Added new autorole: {{role.name}} ({{role.id}}). Points required: {{points}}",
+	"autoRoleInvalidLevel": "I could not resolve `{{parameter}}` into a level, you must input either a number (for example: `1500` or `51000`) or a number suffixed with `l` or `L` (for example: `10L`, `20l`)",
+	"autoRoleInvalidNegativeOrZeroLevel": "I could not resolve `{{parameter}}` into a valid level, they cannot be zero nor negative.",
+	"autoRoleTooSmall": "I could not resolve `{{parameter}}` into a valid level, it resolves to {{points, number}} points, which is below {{minimum, number}}.",
+	"autoRoleTooBig": "I could not resolve `{{parameter}}` into a valid level, it resolves to {{points, number}} points, which is above {{maximum, number}}.",
```

```suggestion
	"autoRoleTooLow": "I could not resolve `{{parameter}}` into a valid level, it resolves to {{points, number}} points, which is below {{minimum, number}}.",
	"autoRoleTooHigh": "I could not resolve `{{parameter}}` into a valid level, it resolves to {{points, number}} points, which is above {{maximum, number}}.",
```

big and small are not how you describe minimum and maximum when it comes to numbers. Big and small are used to describe groups of things, or the length of an object or animal/person.
Update language keys accordingly.